docs/usage/applications.rst

```diff
@@ -65,6 +65,28 @@ connection, around.
     :language: python
     :caption: Handling a database connection
 
+Order of execution
+-----------------------------------------------------
+
+When multiple lifespan context managers and ``on_shutdown`` hooks are specified Litestar will invoke the context
+managers in inverse order before the shutdown hooks are invoked.
+
+Let's say there are two lifespan context managers ``ctx_a`` and ``ctx_b`` as well as two shutdown hooks ``hook_a`` and
```

Comment:
```suggestion
Consider the case where there are two lifespan context managers ``ctx_a`` and ``ctx_b`` as well as two shutdown hooks ``hook_a`` and
```
I think this is easier for ESL people

Reply:
Great point. I am ESL, too, so I am never too sure about how it sounds to native speakers or how digestible it is for others ;)